File: src/main/java/moon/star/ReverseString.scala
package moon.star

import scala.collection.mutable.ListBuffer

class ReverseString {
  def reverseString(s:String):String={
    val str=new StringBuffer()
    var length=s.length-1
    while(length>=0){
      str.append(s(length).toString)
      length-=1
    }
    str.toString
  }
}

File: data-structure/src/main/java/string/ReverseString.java
package string;

public class ReverseString {
    public void solve(char[] s) {
        int i=0,j=s.length-1;
        char a;
        while(i<=j){
            a=s[i];
            s[i]=s[j];
            s[j]=a;
            i++;
            j--;
        }
    }
}
